scripts/startTriggerService.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+export _JAVA_OPTIONS="-Xms8m -Xmx128m"
+
+set -e
+
+TRIGGER_SERVICE_HOST="127.0.0.1"
+TRIGGER_SERVICE_PORT=8088
+
+cleanup() {
+    pids=$(jobs -p)
+    echo Killing $pids
+    [ -n "$pids" ] && kill $pids
+}
+
+waitForTriggerService() {
+    until curl ${TRIGGER_SERVICE_HOST}:${TRIGGER_SERVICE_PORT}/livez
+    do
+        sleep 0.1
```

Comment:
Although `sleep` does not throw an error, based on the docs I've found it accepts an integer.
```suggestion
        sleep 1
```

Reply:
Okay